runtime/cmake/options.cmake

```diff
@@ -123,6 +123,7 @@ option(J9VM_MODULE_WINDBG "Enables compilation of the windbg module.")
 option(J9VM_MODULE_ZIP "Enables compilation of the zip module.")
 option(J9VM_MODULE_ZLIB "Enables compilation of the zlib module.")
 
+option(J9VM_OPT_CRAC_SUPPORT "Enables support for CRAC Java API's")
 option(J9VM_OPT_CRIU_SUPPORT "Enables support for CRIU Java API's")
```

Comment:
The apostrophes are not appropriate here, please remove them:
```
option(J9VM_OPT_CRAC_SUPPORT "Enables support for CRAC Java APIs")
option(J9VM_OPT_CRIU_SUPPORT "Enables support for CRIU Java APIs")
```

Reply:
Removed